src/Pages/Dashboard.js

```diff
@@ -1,12 +1,16 @@
-import React, { Component, useEffect, useState } from 'react'
```

Comment:
We should clean up all these import statements to have some consistent ordering between all files

Reply:
will do!